src/external/sendOtp.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+const sendOtp = async (email, otp) => {
+  try {
+    const awsApi = process.env.REACT_APP_AWS_API;
+    const response = await axios.post(awsApi, { toEmail: email, otp: otp });
+
+    if (response.status !== 200) {
+      throw new Error(`Failed to send OTP: ${response.status}`);
+    }
+
+    // Check if MessageId is present in the response data
+    if (!response.data.MessageId) {
+      throw new Error("Failed to send OTP: No MessageId in response");
+    }
+
+    return true;
+  } catch (error) {
+    throw new Error(error);
```

Comment:
Rather than throwing, we can log the error and return false.

In this method https://github.com/TitansFinance/gmx-interface-fork/pull/9/files#diff-1243dbc064ef4a721fe4e09ca15e8953997665852a67e8b23fb47e94a91f11deR382, we can remove the try / catch and just check for true/false return output from this `sendOtp` method to switch conditions.

Reply:
This is low priority, we do not need to change this now.

Reply:
resolved this in #10, shows a toast error for an incorrectly entered email